react/features/filmstrip/components/Filmstrip.web.js

```diff
@@ -79,6 +132,43 @@ class Filmstrip extends Component {
             </div>
         );
     }
+
+    /**
+     * If the current hover state does not match the known hover state in redux,
+     * dispatch an action to update the known hover state in redux.
+     *
+     * @private
+     * @returns {void}
+     */
+    _notifyOfHoveredStateUpdate() {
```

Comment:
Is there any reason why we don't use the component state for the isHovered variable and dispatch on componentDidUpdate?

Reply:
To avoid re-renders that don't do anything I don't use this.state. This is also an elaborate workaround for erroneous mouseleave events getting fired when hovering over thumbnails.

Reply:
I should also note that mouse events fire very often so this was my workaround for not firing a lot of updates.